examples/mp4.rs

```diff
@@ -76,21 +79,30 @@ fn start_example_client_code() -> Result<()> {
             },
             "encoder_preset": "medium",
             "initial": {
-                "type": "shader",
-                "id": "shader_node_1",
-                "shader_id": "shader_example_1",
-                "children": [
-                    {
-                        "id": "input_1",
-                        "type": "input_stream",
-                        "input_id": "input_1",
-                    }
-                ],
-                "resolution": { "width": VIDEO_RESOLUTION.width, "height": VIDEO_RESOLUTION.height },
+                "id": "input_1",
+                "type": "input_stream",
+                "input_id": "input_1",
```

Comment:
1. It's easier to test synchronization without this weird shader.
2. I'll change examples soon anyway